usr/lib/soft_stdll/soft_specific.c

```diff
@@ -97,8 +97,8 @@ static const MECH_LIST_ELEMENT soft_mech_list[] = {
 #endif
 /* Begin code contributed by Corrent corp. */
 #if !(NODH)
-    {CKM_DH_PKCS_DERIVE, {512, 2048, CKF_DERIVE}},
-    {CKM_DH_PKCS_KEY_PAIR_GEN, {512, 2048, CKF_GENERATE_KEY_PAIR}},
+    {CKM_DH_PKCS_DERIVE, {512, 8192, CKF_DERIVE}},
+    {CKM_DH_PKCS_KEY_PAIR_GEN, {512, 8192, CKF_GENERATE_KEY_PAIR}},
```

Comment:
I guess openssl v1.1 and 3.x support 8192 ...

Reply:
yes